server/server.go

```diff
@@ -340,6 +371,53 @@ func cutReleaseCommandF(args []string, w http.ResponseWriter, slashCommand *MMSl
 	return nil
 }
 
+func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag, repo string) error {
+	var pluginTag = regexp.MustCompile("v[0-9]+.[0-9]+.[0-9]+$")
```

Comment:
@cpanato Where do we confirm whether the user should have access to the slash command?

Reply:
thanks for the catch